src/sdk/modules/validators/k1Validator.test.ts

```diff
@@ -70,57 +70,84 @@ describe("modules.k1Validator.write", async () => {
     expect(balanceAfter - balanceBefore).toBe(1n)
   })
 
-  test.skip("should install k1 validator with 1 owner", async () => {
+  test("should install k1 validator with 1 owner", async () => {
     const isInstalledBefore = await nexusClient.isModuleInstalled({
       module: {
         type: "validator",
         address: addresses.K1Validator,
-        context: "0x"
+        data: encodePacked(["address"], [account.address])
       }
     })
+    console.log(isInstalledBefore, "isInstalledBefore");
 
     if (!isInstalledBefore) {
       const hash = await nexusClient.installModule({
         module: {
           address: addresses.K1Validator,
           type: "validator",
-          context: "0x"
+          data: encodePacked(["address"], [account.address])
```

Comment:
I think we could expose a `getData` helper from each decorator, because it's likely that the data will be likely very smart-account specific. I'd like for each of our core decorator logic to work with other smart accounts, and a 'getData' helper to construct the relevant information might help with that, so that the main logic remains account agnostic. In this case `const getData = (accountAddress: Address) => encodePacked(["address"], [account.address]);` should be exported from the installModule file, and imported here in the test.